### PR TITLE
VueUi based voting

### PR DESCRIPTION
**Pull request by Karolis2011**
tl;dr;
https://cdn.discordapp.com/attachments/131489773268238336/575610491993456640/1557306577_JXnV.mp4

Reworks SSvote to use vueui and display votes live! Without having to spam click to keep refreshing ui.

### CODE REVIEW

**Comment by Karolis2011** [comment]
!review

**Comment by TehFlaminTaco** [comment]
Will this speed up how long it takes for us Australians to see the vote Dialog?

**Comment by Karolis2011** [comment]
@TehFlaminTaco compiled assets of VueUi is about 420 KB, that is sent only once, Bootstrap code is 2 KB. I doubt that it will take a while to open due to slow internet speeds.

File: code/controllers/subsystems/processing/vueui.dm [hunk 1]
```diff
@@ -87,6 +88,20 @@ Byond Vue UI framework's management subsystem
...
+  * @param src_object - object that hosts the uis that we want to close
+  *
+  * @return number of uis closed
+  */
+/datum/controller/subsystem/processing/vueui/proc/close_uis(var/src_object)
+	. = 0
+	for (var/tui in get_open_uis(src_object))
+		var/datum/vueui/ui = tui
+		ui.close()
+	.++
```
Comment (skull132) [inline code/controllers/subsystems/processing/vueui.dm]:
```suggestion
		.++
```

Comment (skull132) [reply code/controllers/subsystems/processing/vueui.dm]: Otherwise we always return 1.

**Pull request merged**